README.md

```diff
@@ -101,6 +101,34 @@ use MonsieurBiz\SyliusSettingsPlugin\Provider\SettingsProviderInterface;
     }
 ```
 
+### Use fixtures
+
+We've implemented a fixtures loader to help you to create your settings if you need to have different settings for your
+tests or project (by channel, by locale…).
+
+You need to create a yaml file with your fixtures, like explained in the documentation of Sylius.  
+You can find our own example in the source code, section `sylius_fixtures`: [configuration file](dist/config/packages/monsieurbiz_settings_plugin_custom.yaml).
+
+It's also possible to run test fixtures with a local suite in development: `make sylius.fixtures.local`.
+
+By default, a fixture will replace the value of a setting if it already exists. 
+If you want to keep a value as it is in the database when running this fixture, you can use the flag `ignore_if_exists: true` for each element that you want to be kept.
+
+### Use CLI
+
+You can use a CLI command to set a value for a setting directly from the console:
+`$ ./bin/console monsieurbiz:settings:set {alias} {path} {type} {value} --channel="FASHION_WEB" --locale="en_US"`
+
+Examples:
+```bash
+$ ./bin/console monsieurbiz:settings:set app.default demo_message text 'fashion message' --channel="FASHION_WEB" --locale="en_US"
+$ ./bin/console monsieurbiz:settings:set app.default demo_message json '{"foo":"baz"}' --channel="FASHION_WEB" --locale="en_US"
+$ ./bin/console monsieurbiz:settings:set app.default demo_message datetime '2023-07-24 01:02:03' --channel="FASHION_WEB" --locale="en_US"
+$ ./bin/console monsieurbiz:settings:set app.default enabled boolean 0
+```
```

Comment:
**❔ question:** ‏If the type of the setting is already set, with an existing setting as example, could we omit it as well?